src/engine/qcommon/cmd.cpp

```diff
@@ -347,8 +347,11 @@ static const char *EscapeString( const char *in, bool quote )
 			// no need to quote semicolons if in ""
 			quoted = true;
 			if ( quote ) break;
+            DAEMON_FALLTHROUGH;
```

Comment:
Don't you need only the first FALLTHROUGH?

Reply:
Yes, you are correct.